### DISCUSSION TITLE
ALERT!

### DISCUSSION THREAD

**Discussion by bangbangyouall** [lightbeam24/CustomTube#234] [General]
this extension is defunct as of reading this, because startube is being worked on (customtube 2) by lightbeam (although he might release 1.3 as is). startube is better, though, because it has better accuracy, better layouts & even is a userscript on V3! now in beta! go to vorapis.pages.dev for V3 & go to https://greasyfork.org/en/scripts/485622-startube to get the script!

**Comment by uy813** [comment]
it didn't change my youtube. Nothing happened.

**Reply by MesaBlack** [reply]
skill issue?

**Reply by bangbangyouall** [reply]
what do you MEAN it didnt work?

**Reply by bangbangyouall** [reply]
oh i see should've installed violentmonkey
